package.json

```diff
@@ -6,12 +6,12 @@
   },
   "license": "MPL-2.0",
   "devDependencies": {
-    "dynalite": "^2.2.0",
-    "serverless": "^1.27.3",
+    "dynalite": "2.3.2",
```

Comment:
The versions here were changed to strict versions to make the local and build environments less varying.